phoenix-core/src/it/java/org/apache/phoenix/end2end/StatsCollectorWithSplitsAndMultiCFIT.java

```diff
@@ -176,7 +176,7 @@ public void testSplitUpdatesStats() throws Exception {
         Collection<GuidePostsInfo> infos = TestUtil.getGuidePostsList(conn, STATS_TEST_TABLE_NAME);
         for (GuidePostsInfo info : infos) {
             assertEquals(20, info.getRowCount());
-            assertEquals(11020, info.getByteCount());
+            assertEquals(11040, info.getByteCount());
```

Comment:
I believe it's because our empty column value isn't empty any longer. We need it to have a value so that Tephra doesn't treat it as a delete marker. We could conditionally use an empty value if the table is non transactional, but meh.